common/src/cformat.rs

```diff
@@ -754,6 +754,12 @@ impl CFormatString {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq)]
```

Comment:
along with YunWon's suggestion, I'd document this. We currently don't do this as much as we'd probably like but it's relatively straight-forward here.